diff --git a/node_modules/@kickstartds/jsonschema-utils/lib/helpers.js b/node_modules/@kickstartds/jsonschema-utils/lib/helpers.js
index bcbaa6a..06541d9 100644
--- a/node_modules/@kickstartds/jsonschema-utils/lib/helpers.js
+++ b/node_modules/@kickstartds/jsonschema-utils/lib/helpers.js
@@ -428,7 +428,7 @@ export function getSortedSchemas(graph) {
 }
 export const defaultProcessingOptions = {
     typeResolution: true,
-    modules: ['base', 'blog', 'form', 'content'],
+    modules: ['base', 'blog', 'form'],
     additionalProperties: 'forceFalse',
     loadPageSchema: true,
     mergeAllOf: true,
